lib/profiling/Makefile

```diff
@@ -40,28 +40,17 @@ LDFLAGS=
 
 # We don't build dl_timer, drhook and nvidia since they require
 # external libraries to be available.
-ALL_LIBS=template simple_timing
+ALL_LIBS=template simple_timing lfric
```

Comment:
Please could you update the copyright date to "2020-2021" on this file.

Reply:
I believe this should be 2019-2021 ... done.